keithradford/ttfm-plus
# fix: auto-awesome functionality to work when tabbed out

### What?

Closes #15. This should fix auto-awesome functionality by allowing it to run in the background. This also removes the `eventPage` typescript file as it has no use.

### How to test it?

1. Run `npm run build` on this branch
2. Load the unpacked `dist` into `chrome://extensions` (make sure Developer mode is enabled)
3. Navigate to turntable and ensure auto-awesome is working when tabbed in, tabbed out, or on a different tab.
